ponyo/generate_labeled_data.py

```diff
@@ -19,9 +19,11 @@
 from sklearn import preprocessing
 
 import warnings
-warnings.filterwarnings(action='ignore')
+
+warnings.filterwarnings(action="ignore")
```

Comment:
You may want to leave a comment with this line to note which warnings are supposed to be ignored, or set a more precise warnings filter. It's possible that you will end up missing something important by suppressing all warnings indiscriminately.

Reply:
Good point. I have added a specific warning for deprecated functions since the env that ponyo is using is from the analysis repos (simulate-expression-compendia and generic-expression-patterns) which have some fixed versions. I'm new to developing software so perhaps there is better way to handle versioning